api-server/web-server/src/api/v2.rs

```diff
@@ -620,11 +616,31 @@ pub async fn address<T: ApiServerStorage>(
         })?
         .unwrap_or(Amount::ZERO);
 
+    let mut tokens = Vec::with_capacity(address_balances.len());
+    let mut coin_balance = AmountWithDecimals {
+        amount: Amount::ZERO,
+        decimals: state.chain_config.coin_decimals(),
+    };
+
+    for (coin_or_token_id, balance) in address_balances {
+        match coin_or_token_id {
+            CoinOrTokenId::Coin => {
+                coin_balance = balance;
+            }
+            CoinOrTokenId::TokenId(token_id) => {
+                tokens.push(json!({
+                    "token_id": Address::new(&state.chain_config, token_id).expect("no error").as_str(),
+                    "amount": amount_to_json(balance.amount, balance.decimals),
+                }));
+            }
+        }
+    }
+
     Ok(Json(json!({
-    "coin_balance": amount_to_json(coin_balance, state.chain_config.coin_decimals()),
-    "locked_coin_balance": amount_to_json(locked_coin_balance, state.chain_config.coin_decimals()),
-    "transaction_history": transaction_history
-    //TODO "token_balances": destination_summary.token_balances(),
+        "coin_balance": amount_to_json(coin_balance.amount, coin_balance.decimals),
+        "locked_coin_balance": amount_to_json(locked_coin_balance, state.chain_config.coin_decimals()),
+        "transaction_history": transaction_history,
+        "tokens": tokens
     })))
 }
```

Comment:
Btw, could you fix similar formatting issues in other places in this file (in this or other PR)? Namely in the functions `block`, `chain_tip`, `transaction_merkle_path`. Initially I wanted to fix them myself in a small cleanup PR, but I didn't want to create conflicts.